### DISCUSSION TITLE
Ignore case with v12 of PrepareHeaderForMatch in Visual Basic

### DISCUSSION THREAD

**Discussion by jorgeshimada** [JoshClose/CsvHelper#1860] [General]
I'm trying to implement to ignore the header name case, using CSVHelper 12 in Visual Basic.
The code below didn't work. Can every explain or show me how to do it 

Using csv As New CsvReader(Reader)
	csv.Configuration.PrepareHeaderForMatch = (String header) >= header.ToUpper()
End Using

Any help will be appreciate. Thanks in advance

**Comment by AltruCoder** [comment]
Try this
```vb
csv.Configuration.PrepareHeaderForMatch = Function(header, field) header.ToUpper()
```

**Reply by jorgeshimada** [reply]
It worked well! Thank you so much! Your tip helped me very much!
